src/main/java/edu/wpi/first/pathweaver/CreateProjectController.java

```diff
@@ -54,6 +89,16 @@ private void initialize() {
     ObservableList<TextField> allFields = FXCollections.observableArrayList(numericFields);
     allFields.add(directory);
 
+    var timeControls = List.of(timeLabel, timeStep, timeUnits);
+    var velocityControls = List.of(velocityLabel, maxVelocity, velocityUnits);
+    var accelerationControls = List.of(accelerationLabel, maxAcceleration, accelerationUnits); // NOPMD
+    var jerkControls = List.of(jerkLabel, maxJerk, jerkUnits);
+    var wheelBaseControls = List.of(wheelBaseLabel, wheelBase, wheelBaseUnits);
+
+    var allTooltipControls = new ArrayList<Control>(); // NOPMD
```

Comment:
This could be inlined as a `Stream` operation where it's used later in the method